cmd/restic/cmd_check.go

```diff
@@ -365,6 +361,12 @@ func runCheck(ctx context.Context, opts CheckOptions, gopts global.Options, args
 		return summary, ctx.Err()
 	}
 
+	opts.filteredStatus = chkr.FilterStatus()
```

Comment:
Let's move it below the `if opts.CheckUnused {` block. Then the call is directly next to the place where it's used.